Sourcery/Sourcery.swift

```diff
@@ -349,7 +350,8 @@ extension Sourcery {
         }, finallyBlock: {})
 
         result = try processFileRanges(for: parsingResult, in: result)
-        return try processInlineRanges(for: parsingResult, in: result)
+        result = try processInlineRanges(for: parsingResult, in: result)
+        return TemplateAnnotationsParser.removingEmptyAnnotations(from: result)
```

Comment:
These three lines is an exact duplication of the line in the guard above, maybe we can put them somewhere to have only in one place? 😄